acapy_plugin_toolbox/static_connections.py

```diff
@@ -181,16 +181,20 @@ class StaticConnectionGetListHandler(BaseHandler):
         wallet: BaseWallet = await context.inject(BaseWallet)
         try:
             tag_filter = dict(
+                filter(lambda item: item[1] is not None, {
+                    'my_did': context.message.my_did,
+                    'their_did': context.message.their_did,
+                }.items())
+            )
+            post_filter = dict(
                 filter(lambda item: item[1] is not None, {
                     'initiator': context.message.initiator,
                     'invitation_key': context.message.invitation_key,
-                    'my_did': context.message.my_did,
                     'invitation_mode': ConnectionRecord.INVITATION_MODE_STATIC,
-                    'their_did': context.message.their_did,
                     'their_role': context.message.their_role
                 }.items())
             )
-            records = await ConnectionRecord.query(context, tag_filter)
+            records = await ConnectionRecord.query(context, tag_filter, post_filter)
```

Comment:
Why does ConnectionRecord still have tag_filter?

Reply:
https://github.com/hyperledger/aries-cloudagent-python/blob/master/aries_cloudagent/connections/models/connection_record.py#L27-L32

They retained tags for connection records for my_did and their_did.
The only values retained in tags for the other records were the thread_ids, which aren't as useful for us.

Reply:
Thanks for the explanation.